.github/release-template.md

```diff
@@ -0,0 +1,34 @@
+Darwinia Parachain
+===
+
+## [5.0.1] - 2022-01-27
+
+> :warning: **This release introduces a new host function. Please upgrade your node prior to the next runtime upgrade of Crab Parachain or Darwinia Parachain in order for your node to continue syncing.**
+
+|      Network       | Native Runtime | Upgrade Priority |
+| :----------------: | :------------: | :--------------: |
+| Darwinia Parachain |       2        |       HIGH       |
+|  Crab  Parachain   |       2        |       HIGH       |
+
+## Resources
+
+### Pre-built Binary
+|  OS   |  Arch  | Glibc | LLVM  |                                                                 Download                                                                 |
+| :---: | :----: | :---: | :---: | :--------------------------------------------------------------------------------------------------------------------------------------: |
+| Linux | x86_64 | 2.23  |  4.0  | [tar.bz2](https://github.com/darwinia-network/darwinia-parachain/releases/download/{{ tag }}/darwinia-collator-x86_64-linux-gnu.tar.bz2) |
```

Comment:
Please check this.

Reply:
https://github.com/fewensa/darwinia-parachain/releases/download/v5.0.3/darwinia-collator-x86_64-linux-gnu.tar.bz2

https://github.com/darwinia-network/darwinia-parachain/releases/download/{{ tag }}/darwinia-collator-x86_64-linux-gnu.tar.bz2

It's  right